Tests/SuperLinq.Test/FallbackIfEmptyTest.cs

```diff
@@ -10,6 +10,22 @@ public void FallbackIfEmptyWithEmptySequence()
 		source.FallbackIfEmpty(12, 23).AssertSequenceEqual(12, 23);
 	}
 
+	[Fact]
+	public void FallbackIfEmptyWithEmptyListSequence()
+	{
+		var source = Enumerable.Empty<int>().ToList();
+		source.FallbackIfEmpty(12).AssertSequenceEqual(12);
+		source.FallbackIfEmpty(12, 23).AssertSequenceEqual(12, 23);
+	}
+
+	[Fact]
+	public void FallbackIfEmptyWithCollectionListSequence()
```

Comment:
Add a matching test with a non-empty collection, which passes in a `BreakingSequence`.